.github/workflows/update-sync-param-files.yaml

```diff
@@ -0,0 +1,35 @@
+name: update-sync-param-files
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  sync-param-files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate-token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+
+      - uses: actions/checkout@v1
+
+      - name: Generate sync-param-files.yaml
+        run: |
+          git clone https://github.com/autowarefoundation/autoware.universe.git -b main /tmp/autoware.universe
+          python3 .github/workflows/update-sync-param-files.py .github/sync-param-files.yaml
+
+      - name: Create PR
+        id: create-pr
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          base: ${{ github.event.repository.default_branch }}
+          branch: update-sync-param-files
+          title: "chore: update sync-param-files.yaml"
+          commit-message: "chore: update sync-param-files.yaml"
+          body: ${{ steps.create-pr-body.outputs.body }}
```

Comment:
Please consider adding these steps for better maintainability.
https://github.com/autowarefoundation/autoware/blob/e108b8fa9e47307b1f2573be5d0dde828235bcb0/.github/workflows/update-tool-versions.yaml#L54-L66

Reply:
Please check this commit
https://github.com/autowarefoundation/autoware_launch/pull/75/commits/bd49be9fdc1248a9bd4c775e5e8769e9af44ea1c